File: Elixir/1005.ex
{a, _} = IO.gets("") |> String.trim |> Float.parse
{b, _} = IO.gets("") |> String.trim |> Float.parse
IO.puts("MEDIA = #{:erlang.float_to_binary((a * 3.5 + b * 7.5) / 11, [decimals: 5])}")

File: Rust/1005.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let a: f64 = input.trim().parse().unwrap();

    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let b: f64 = input.trim().parse().unwrap();

    println!("MEDIA = {:.5}", (a * 3.5 + b * 7.5) / 11.0);
}